backend/ebios_rm/helpers.py

```
@@ -2,7 +2,14 @@
 import math
 import random
 from global_settings.models import GlobalSettings
-from .models import EbiosRMStudy, FearedEvent, RoTo
+from .models import (
+    AttackPath,
+    EbiosRMStudy,
```

Ruff (0.8.2)

`.models.EbiosRMStudy` imported but unused

Remove unused import: `.models.EbiosRMStudy`

(F401)